akka-http-core/src/main/scala/akka/http/impl/settings/ClientConnectionSettingsImpl.scala

```diff
@@ -24,7 +24,8 @@ private[akka] final case class ClientConnectionSettingsImpl(
   requestHeaderSizeHint: Int,
   websocketRandomFactory: () ⇒ Random,
   socketOptions: immutable.Seq[SocketOption],
-  parserSettings: ParserSettings)
+  parserSettings: ParserSettings,
+  redirectSettings: akka.http.scaladsl.settings.ClientAutoRedirectSettings)
```

Comment:
AFAICS keeping binary compatibility will force us to write a constructor the same as the previous one with a default value for the new parameter here.